### PR TITLE
Updating info about storage

### PR DESCRIPTION
**Pull request by dockerymick**
https://app.zenhub.com/workspaces/engineering-backlog-do-not-delete-604fab62d4b98d00150a2854/issues/stolostron/backlog/21137

### CODE REVIEW

File: observability/observe_environments.adoc [hunk 1]
```diff
@@ -220,7 +220,7 @@ Observability components require 2701mCPU and 11972Mi memory to install the obse
 [#persistent-stores-observability]
 == Persistent stores used in the observability service
 
-When you install {product-title-short} the following persistent volumes are created:
+When you install {product-title-short} the following persistent volumes (PV) must be created so that Persistent Volume Claims (PVC) can attach to it automatically. As a reminder, you must define a storage class in the `MultiClusterObservability` CR when there is o default storage class specified. It is recommended to use Block Storage, similare to what Prometheus uses. Also each replica of `alertmanager`, `thanos-compactor`, `thanos-ruler`, `thanos-receive-default` and `thanos-store-shard` must have its own PV.
```
Comment (bjoydeep) [inline observability/observe_environments.adoc]: A minor comments @dockerymick :
`As a reminder, you must define a storage class in the `MultiClusterObservability` CR when there is o default storage class specified.` 
Can we rephrase this to read:
`As a reminder, you must define a storage class in the `MultiClusterObservability` CR when there is no default storage class specified or you want to use a non-default storage class to host the PVs.`

**Comment by dockerymick** [comment]
Thanks for your review @bjoydeep ! I have updated the file. Moving forward with this merge
